### PR TITLE
Added frequency calculation for bitfury miner

### PR DESCRIPTION
**Pull request by legkodymov**
- Changed copyright for tm_i2c.c.
- Added frequency calculation for bitfury miner.

### CODE REVIEW

File: driver-bitfury.c [hunk 1]
```diff
@@ -653,7 +680,7 @@ void bitfury_do_io(struct thr_info * const master_thr)
 		}
 	}
 	
-	timer_set_delay_from_now(&master_thr->tv_poll, 10000);
+	timer_set_delay_from_now(&master_thr->tv_poll, 200000);
```
Comment (luke-jr) [inline driver-bitfury.c]: Is slowing this down necessary?

Comment (legkodymov) [inline driver-bitfury.c]: Not necessary. But doing more than 2 polls per "job period" is not necessary. Even 1.1-1.9 polls is enough. I know, p2pool is special case.
I guess it needs special automatic calculation in future or config option.

Comment (luke-jr) [inline driver-bitfury.c]: Even normal pools benefit from more frequent polling (fewer stale shares), just much less. If there's no benefit to polling less often, it probably makes sense to just leave it every 10ms?

File: driver-bitfury.c [hunk 2]
```diff
@@ -679,6 +706,7 @@ struct api_data *bitfury_api_device_status(struct cgpu_info * const cgpu)
 	int clock_bits = bitfury->osc6_bits;
 	
 	root = api_add_int(root, "Clock Bits", &clock_bits, true);
+	root = api_add_double(root, "Mhz", &bitfury->mhz, false);
```
Comment (luke-jr) [inline driver-bitfury.c]: Please change this to:

``` C
root = api_add_freq(root, "Frequency", &bitfury->mhz, false);
```

File: driver-bitfury.c [hunk 3]
```diff
@@ -599,7 +602,31 @@ void bitfury_do_io(struct thr_info * const master_thr)
...
-		
+
+		counter = bitfury_decnonce(newbuf[n]);
+		if ((counter & 0xFFC00000) == 0xdf800000)
+		{
+			unsigned int cycles;
+			long long unsigned int period;
+			double ns;
+			struct timeval d_time;
+			static int skip;
```
Comment (luke-jr) [inline driver-bitfury.c:613]: Sorry, I missed this: why is it static?

**Comment by luke-jr** [comment]
Merged as 75a7e14

**Pull request closed without merging**